include/keyence/impl/high_speed_defs.h

```diff
@@ -3,9 +3,28 @@
 
 #include <stdint.h>
 
+#define KEYENCE_FUNDAMENTAL_LENGTH_UNIT (0.01 * 1.0e-6)
+// The integer encoded values representing 'out of range'
+// received from the sensor
+#define KEYENCE_INFINITE_DISTANCE_VALUE -524288
```

Comment:
OK.  Can we add some comments to clarify this in the code (including the items that both @gavanderhoorn and @Jmeyer1292 discussed).